pkg/reconciler/v1alpha1/taskrun/taskrun.go

```diff
@@ -156,7 +156,7 @@ func (c *Reconciler) Reconcile(ctx context.Context, key string) error {
 
 func (c *Reconciler) reconcile(ctx context.Context, tr *v1alpha1.TaskRun) error {
 	// get build the same as the taskrun, this is the value we use for 1:1 mapping and retrieval
-	build, err := c.buildLister.Builds(tr.Namespace).Get(tr.Name)
+	build, err := c.BuildClientSet.BuildV1alpha1().Builds(tr.Namespace).Get(tr.Name, metav1.GetOptions{})
 	if errors.IsNotFound(err) {
```

Comment:
if the buildLister is not needed anymore, can we remove it?

Reply:
Did we decide if we want to use the client set here? I think we want to keep using the lister b/c it has cacheing?

Reply:
Oh i see, there is a discussion below